site/_posts/2020-07-15-octant-0.14-release.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Octant 0.14 - Now with Resource Creation"
+excerpt: Octant now supports the ability to create resources for 0.16+ Kubernetes clusters by taking advantage of server side apply.
```

Comment:
`by taking advantage of server side apply` can be removed since it is an excerpt